cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -203,6 +203,16 @@
         "ubuntu1804"
       ]
     },
+    "Arch": {
```

Comment:
I'd call the parameter Architecture or OSArchitectur, but if possible I'd concatenate it to the ami parameter or another of the ones parameter.

Reply:
Done. I'll eventually probably tack it onto the master instance type paramter.